providers/implementations/encode_decode/encode_key2text.c

```diff
@@ -130,7 +131,8 @@ static int print_labeled_bignum(BIO *out, const char *label, const BIGNUM *bn)
 /* Number of octets per line */
 #define LABELED_BUF_PRINT_WIDTH    15
 
-#if !defined(OPENSSL_NO_DH) || !defined(OPENSSL_NO_DSA) || !defined(OPENSSL_NO_EC)
+#if !defined(OPENSSL_NO_DH) || !defined(OPENSSL_NO_DSA) \
+    || !defined(OPENSSL_NO_EC) || !defined(OPENSSL_NO_SLH_DSA)
 static int print_labeled_buf(BIO *out, const char *label,
                              const unsigned char *buf, size_t buflen)
```

Comment:
This function has moved. The branch should be rebased on master, and then the PR on the updated branch.

Reply:
Could you rebase the branch on master please?

Reply:
The branch has been rebased.